modules/components/analytics/core/kpi/org.wso2.carbon.pc.analytics.core.kpi/src/main/java/org/wso2/carbon/pc/analytics/core/kpi/clients/LoginAdminServiceClient.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.pc.analytics.core.kpi.clients;
+
+/**
+ * Connects with DAS's LoginAdminService
+ */
+
+import org.apache.axis2.AxisFault;
+import org.apache.axis2.transport.http.HTTPConstants;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.authenticator.stub.AuthenticationAdminStub;
+import org.wso2.carbon.authenticator.stub.LoginAuthenticationExceptionException;
+import org.wso2.carbon.authenticator.stub.LogoutAuthenticationExceptionException;
+import org.apache.axis2.context.ServiceContext;
+import org.wso2.carbon.pc.analytics.core.kpi.AnalyticsConfigConstants;
+
+import java.rmi.RemoteException;
+import java.util.Arrays;
+
+public class LoginAdminServiceClient {
+    private final String serviceName = AnalyticsConfigConstants.AUTHENTICATION_ADMIN;
```

Comment:
Thanks @mohanvive. Yes it is not needed. Fixed with 7bbf8fb2eea6315ebb8b4104c667fd68e7212046